### DISCUSSION TITLE
What type of account to create

### DISCUSSION THREAD

**Discussion by Muzilcy** [community/community#111183] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello, I want to create an account of the same type as SuperMap, but I don't know if it is specifically an organization/team or business account?
https://github.com/SuperMap

**Comment by koeppel** [comment]
Hi,

SuperMap is an Organization. You can see this by clicking on the "Repositories" tab it will change the URL to https://github.com/orgs/SuperMap/repositories

AFAIK a "Team" is simply the wording of a premium version of an Organization.

**Reply by Muzilcy** [reply]
If I start with a free organization, can I upgrade to a paid team later？

**Reply by koeppel** [reply]
Yes of course :) I think you can always upgrade / downgrade your accounts

**Reply by koeppel** [reply]
If this answer helped you, please mark it as the answer of the question

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Muzilcy ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!
